learning_resources/etl/utils.py

```diff
@@ -564,6 +573,8 @@ def generate_course_numbers_json(
                     if dept_id
                     else None
                 ),
+                "sort_coursenum": sort_coursenum,
```

Comment:
Adding these fields to the `course.course_numbers` field so they can be used by the database REST API too